Resources/config/services.yml

```diff
@@ -11,16 +11,21 @@ services:
             - '@assets.packages'
 
     gluggi.finder:
```

Comment:
Is this service intended to be a publicly used or is it more of an internal one?